src/AlgebraicGeometry/ToricVarieties/AlgebraicCycles/constructors.jl

```diff
@@ -296,7 +296,7 @@ function Base.show(io::IO, ac::RationalEquivalenceClass)
       # otherwise, extract properties to represent the rational equivalence class
       r = representative(ac)
       coeffs = [c for c in AbstractAlgebra.coefficients(r)]
-      expos = [matrix(ZZ, [k for k in exponent_vectors(m)]) for m in AbstractAlgebra.monomials(r)]
+      expos = [matrix(ZZ, [k for k in AbstractAlgebra.exponent_vectors(m)]) for m in AbstractAlgebra.monomials(r)]
```

Comment:
With your changes, maybe also the following works?
```suggestion
      expos = [matrix(ZZ, [k for k in exponents(m)]) for m in AbstractAlgebra.monomials(r)]
```

Reply:
Seeing that `m` is a monomial anyway and there is hence no monomial ordering involved, I guess so. `AbstractAlgebra.exponent_vectors` will be faster though, from what I understand.